src/Mobs/Enderman.cpp

```diff
@@ -5,9 +5,60 @@
 #include "Enderman.h"
 #include "../Entities/Player.h"
 #include "../LineBlockTracer.h"
+#include "../BlockInfo.h"
 
+bool FindTeleportDestination(cWorld *a_World, Vector3i a_Centre, const int a_HalfCubeWidth, const int a_HeightRequired, const unsigned int a_NumTries, Vector3i &a_Destination)
+{
+	/*
+	Algorithm:
+	Choose random destination.
+	Seek downwards, regardless of distance until the block is made of movement-blocking material: https://minecraft.fandom.com/wiki/Materials
+	Succeeds if no liquid or solid blocks prevents from standing at destination.
+	*/
+	auto & Random = GetRandomProvider();
+
+	for (unsigned int i=0; i<a_NumTries; i++)
```

Comment:
```suggestion
	for (unsigned int i = 0; i < a_NumTries; i++)
```
Please use space between operators